superset-frontend/src/components/ErrorMessage/ErrorAlert.stories.tsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Meta, StoryFn } from '@storybook/react';
+import { Layout, Row, Col, Card } from 'antd-v5';
+import ErrorAlert from './ErrorAlert';
+
+const { Content } = Layout;
+
+const longDescription = `This is a detailed description to test long content display.
+Line breaks are included here to demonstrate pre-wrap styling.
+This is useful for verbose error messages.`;
+
+const sqlErrorDescription = `SQL Error: Syntax error near unexpected token.
+Please check your query and ensure it follows the correct syntax.`;
+
+const detailsExample = `Additional details about the issue are provided here.
+This content is shown when the user clicks "Show more".`;
+
+const ErrorCard: React.FC<{ children: React.ReactNode }> = ({ children }) => (
+  <Card>{children}</Card>
+);
+
+export default {
+  title: 'Components/ErrorAlert',
+  component: ErrorAlert,
+} as Meta;
+
+export const Gallery: StoryFn = () => (
```

Comment:
This is not coming with this PR but I find it odd that an ErrorAlert can be of type info or warning. Naming here could probably be improved.

Reply:
Right, maybe limiting to warning/error here makes more sense here. I think there may be one or two edge cases where `ErrorAlert` is used to `info`, maybe cause someone wanted some of the features in here. We should probably change those to use a normal `Alert`